services/director-v2/src/simcore_service_director_v2/modules/api_key_resource_manager.py

```diff
@@ -0,0 +1,79 @@
+from functools import cached_property
+from uuid import uuid5
+
+from fastapi import FastAPI
+from models_library.api_schemas_webserver import WEBSERVER_RPC_NAMESPACE
+from models_library.api_schemas_webserver.auth import ApiKeyCreate, ApiKeyGet
```

Comment:
Question: I will be introducing some additional rpc calls to the webserver. Should I add them to the api_schemas_webserver? or should we introduce something like rpc_schemas_websever? I can see that you are reusing the same one.

Reply:
I would say the former `rpc_schemas_websever`. For me I already had these as public API exposed to the outside world.

Reply:
@matusdrobuliak66 this will notify you to read my answer above.